packages/react/src/components/Tooltip/Tooltip.js

```diff
@@ -363,17 +369,14 @@ class Tooltip extends Component {
       onBlur: this.handleMouse,
       'aria-haspopup': 'true',
       'aria-expanded': open,
-      // if the user provides property `triggerText`,
-      // then the button should use aria-describedby to point to its id,
-      // if the user doesn't provide property `triggerText`,
-      // then an aria-label will be provided via the `iconDescription` property.
-      ...(triggerText
-        ? {
-            'aria-describedby': triggerId,
-          }
-        : {
-            'aria-label': iconDescription,
-          }),
+      // if the user provides and aria-label go with that
+      // if the user provides triggerText -- set aria-labelledby to that elements ID
+      // if the user provides neither of those set aria-label to icon-description
+      ...(ariaLabel
```

Comment:
since React supports the hyphenated aria attributes do we want to add the `ariaLabel` prop or just pass through `aria-label`?

also can we extract the logic out to a function and avoid nested ternary?